bin/runLint.js

```diff
@@ -21,8 +21,18 @@ const adpDevsiteUtilsDir = path.dirname(__dirname);
 // Get the current working directory (target repo where the command is run)
 const targetDir = process.cwd();
 
+// Check for flags
+const deadLinksOnly = process.argv.includes('--dead-links-only');
+const skipDeadLinks = process.argv.includes('--skip-dead-links') || process.argv.includes('--no-dead-links');
```

Comment:
We don't want to skip dead link checks when `--no-dead-links`, do we?

```suggestion
const skipDeadLinks = process.argv.includes('--skip-dead-links');
```